### ISSUE TITLE
Research Feedback van Marco 

### ISSUE THREAD

**Issue by marcoFijan** [randy554/functional-programming#3]
**Concept**
Interessant en leuk concept. Hier kun je leuke visualisaties van maken

Ik zou 'gemeente' nog iets meer onderbouwen wat je daarmee bedoeld. Ga je voor alle gemeentes van heel Nederland? Alle gemeentes van een provincie? Gemeentes rondom een grote stad? Of alleen grote gemeentes maar uit verschillende gedeeltes van het land

Ik zou misschien nog iets meer subvragen gaan stellen. Ookal heb je de informatie al gevonden, zoals bijvoorbeeld:
* Wat is het gemiddeld inkomen per gemeente
* Wat is het gemiddelde aan parkeerplekken in Nederland per gemeente
* Hoeveel gratis en hoeveel betaalde parkeerplaatsen zijn er per gemeente

Verder echt een leuk orgineel gevonden onderwerp!

**Comment by gijslaarman** [comment]
Yes! Leuk en origineel idee, is even kijken of we kunnen stellen dat rijke mensen meer ruimte nodig hebben (volgens hen dan he). En inderdaad zoals Marco zegt, maak het concreet welke informatie je nodig hebt (dat zit al redelijk goed verpakt in je onderzoeksvraag). En inderdaad dit bied kansen!
